lib/Perl/Critic/Policy/ControlStructures/ProhibitYadaOperator.pm

```diff
@@ -0,0 +1,95 @@
+package Perl::Critic::Policy::ControlStructures::ProhibitYadaOperator;
+
+use 5.006001;
+use strict;
+use warnings;
+use Readonly;
+
+use Perl::Critic::Utils qw{ :characters :severities };
+use base 'Perl::Critic::Policy';
+
+our $VERSION = '1.123';
+
+#-----------------------------------------------------------------------------
+
+Readonly::Scalar my $DESC => q{yadda operator (...) used};
+Readonly::Scalar my $EXPL => [ 100 ];
+
+#-----------------------------------------------------------------------------
+
+sub supported_parameters { return ()                         }
+sub default_severity     { return $SEVERITY_HIGH             }
+sub default_themes       { return qw( core pbp maintenance ) }
+sub applies_to           { return 'PPI::Token::Operator' }
+
+#-----------------------------------------------------------------------------
+
+sub violates {
+    my ( $self, $elem, undef ) = @_;
+
+    if ( _is_yada( $elem ) ) {
+        return $self->violation( $DESC, $EXPL, $elem );
+    }
+    return;    #ok!
+}
+
+sub _is_yada {
+    my ( $elem ) = @_;
+
+    return unless $elem eq '...';
```

Comment:
The perlcritic rules forbid the use of `unless`. Personally, I don't agree. But we eat our own dogfood her :).

There is a test script somewhere in `xt/` that checks all the code for self-compliance.

Say `./Build authortest` to run it (I think).